plugins/modules/fusion_se.py

```diff
@@ -53,23 +53,28 @@
     type: str
     default: iscsi
     choices: [ iscsi ]
-  network_interface_groups:
```

Comment:
I think these parameters will have to be properly deprecated - see e.g. [pg](https://github.com/Pure-Storage-Ansible/Fusion-Collection/blob/master/plugins/modules/fusion_pg.py#L282) and leave a stub in AnsibleModule params.